src/pages/unlock.tsx

```diff
@@ -65,7 +85,13 @@ export function Unlock(): JSX.Element {
                 : undefined
             }
           >
-            Enter the password you used on this device.
+            {waitingMessage ? (
+              <Caption fontSize={0} mt="base-loose" textAlign={'center'}>
+                {waitingMessage}
+              </Caption>
+            ) : (
+              'Enter the password you used on this device.'
```

Comment:
If we put the ternary inside the caption, so it's always a caption, we'd prevent the layout vertical reshift when it goes from the first to second msg.